### DISCUSSION TITLE
Change Windows Defualt for Dynamic Lighting to off

### DISCUSSION THREAD

**Discussion by initiumx** [ChrisTitusTech/winutil#2690] [Ideas]
Windows 11 has a Dynamic Lighting setting that is auto enabled and locked behind activating Windows. For some peripheral devices this is a nightmare as it effectively locks the personalisation of these devices to the settings in Windows when connecting to a Windows OS pc. Even older devices that had not previously required this lighting sync option seem to be effectively locked into Windows rether than the independent peripheral software.
I now have a serious issue with a LogiTech mouse suddenly becoming locked behind Windows OS and causing me hours of frustration due to the OS on the pc not being 'Activated' when it had worked perfectly fine previous to a firmware update. It even goes so far as to corrupt the settings on the mouse if you open Windows Settings to check. While some may say this is an issue for LogiTech, after digging around and finding that Windows locked the setting to 'on', it just angers me so much that such an outright anti-consumer action is taken by MS. It's my opinion of course, but no peripheral device should be overwritten by MS.

![Windows-Settings-Dynamic-Lighting](https://github.com/user-attachments/assets/da1ab33f-300a-4bce-8963-e54c44355823)

If there is a way to have this setting switched to 'off', rather than 'on' as it is currently, then peripheral devices with lighting settings will be able to return back to the users choice (hopefully). Please see the screen grab to further highlight the issue.

**Comment by Neobond** [comment]
I know this is old, but to override this you simply have to have the Logitech app that controls the lighting installed and manage the lighting from there.

You can see below the greyed out option that "Compatible apps in the foreground always control lighting" is also set to "On" this means that the Logitech app that controls the lighting would override the Dynamic Lighting of Windows.
